file-formats/dmon/examples/z_aff_example_dmon.dmon.json

```
@@ -2,11 +2,12 @@
   "formatVersion": "1",
   "header": {
     "description": "AFF ABAP Daemon Application",
-    "originalLanguage": "en"
+    "originalLanguage": "en",
+    "abapLanguageVersion": "0"
```

```suggestion
    "originalLanguage": "en"
```

Currently the abap language version is "0" which is wrong. The object owner is investigating how it can be fixed.